README.md

```diff
@@ -90,10 +75,11 @@ If you would like to contribute to this pipeline, please see the [contributing g
 
 For further information or help, don't hesitate to get in touch on the [Slack `#hic` channel](https://nfcore.slack.com/channels/hic) (you can join with [this invite](https://nf-co.re/join/slack)).
 
-## Citation
+## Citations
 
-If you use nf-core/hic for your analysis, please cite it using the following
-doi: [10.5281/zenodo.2669513](https://doi.org/10.5281/zenodo.2669513)
+If you use nf-core/hic for your analysis, please cite it using the following doi: doi: [10.5281/zenodo.2669513](https://doi.org/10.5281/zenodo.2669513)
```

Comment:
This is the first record, I think we want to cite this one that will always resolve to the latest, no?
```suggestion
If you use nf-core/hic for your analysis, please cite it using the following doi: doi: [10.5281/zenodo.2669512](https://doi.org/10.5281/zenodo.2669512)
```